conf/sample.ini

```diff
@@ -993,8 +993,14 @@
 ;enabled = true
 
 [geomap]
-# Set the default base layer in GeoMaps plugin
-;default_baselayer =
+# Set the JSON configuration for the default basemap
+;default_baselayer_config = `{
+  "type": "xyz",
+  "config": {
+    "attribution": "Open street map",
+    "url": "https://tile.openstreetmap.org/{z}/{x}/{y}.png"
+  }
+}`
 
 # Enable or disable loading other base map layers
-;disable_custom_baselayers = false
+;enable_custom_baselayers = true
```

Comment:
I flipped this to "enable" since every other property is set in the affirmative.